Type Bug: Incompatible Types

### Description

src/hiero_sdk_python/tokens/token_transfer_list.py:79: error: Incompatible types in assignment (expression has type "TokenNftTransfer", variable has type "TokenTransfer")

### Steps to reproduce

To do this, run mypy path_to/token_transfer_list.py

We should fix this type conflict


### Additional context

_No response_

### Hedera network

_No response_

### Version

v0.1.4

### Operating system

None

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -76,6 +76,7 @@ This changelog is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.
 - Invalid HEX representation and signature validation in keys_public_ecdsa.py
 - Invalid signature verification for examples/keys_public_der.py
 - Duplicate validation function in TokenCreate
+- Incompatible Types assignment in token_transfer_list.py
 
 ### Removed
 - Removed the old `/documentation` folder.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -6,6 +6,11 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](https://semver.org).  
 This changelog is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 
+## [Unreleased]
+
+### Fixed
+- Incompatible Types assignment in token_transfer_list.py
+
 ## [0.1.5] - 2025-09-25
 
 ### Added
@@ -76,7 +81,6 @@ This changelog is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.
 - Invalid HEX representation and signature validation in keys_public_ecdsa.py
 - Invalid signature verification for examples/keys_public_der.py
 - Duplicate validation function in TokenCreate
-- Incompatible Types assignment in token_transfer_list.py
 
 ### Removed
 - Removed the old `/documentation` folder.

--- a/src/hiero_sdk_python/tokens/token_transfer_list.py
+++ b/src/hiero_sdk_python/tokens/token_transfer_list.py
@@ -73,11 +73,11 @@ class TokenTransferList:
             expected_decimals={'value':self.expected_decimals} if self.expected_decimals else None
         )
 
-        for transfer in self.transfers:
-            proto.transfers.append(transfer._to_proto())
+        for fungible_transfer in self.transfers:
+            proto.transfers.append(fungible_transfer._to_proto())
 
-        for transfer in self.nft_transfers:
-            proto.nftTransfers.append(transfer._to_proto())
+        for nft_transfer in self.nft_transfers:
+            proto.nftTransfers.append(nft_transfer._to_proto())
 
         return proto
 

